### ISSUE TITLE
error with evaluation script

### ISSUE THREAD

**Issue by Eugen2525** [sigmorphon/2019#1]
While running evaluation script for task2 I am getting the following error:

```
Traceback (most recent call last):
  File "evaluate_2019_task2.py", line 138, in <module>
    main()
  File "evaluate_2019_task2.py", line 134, in main
    results = manipulate_data(input_pairs(reference, output))
  File "evaluate_2019_task2.py", line 65, in manipulate_data
    for r, o in pairs:
  File "evaluate_2019_task2.py", line 121, in input_pairs
    for r, o in zip(reference, output):
  File "evaluate_2019_task2.py", line 116, in read_conllu
    with open(file) as f:
TypeError: invalid file: PosixPath('sample_2019_task2_gold.txt')

```
I am not sure how to fix it. I am using python3

**Comment by aryamccarthy** [comment]
To evaluate performance, you must pass an _existing_ file with the `--reference` flag. In particular, it should be the dev file for the language you're evaluating. 

You've used `sample_2019_task2_gold.txt` as your reference file, which will only work if (a) that file is in the same directory as the script, so it is found, and (b) you are comparing to the sample system output we provide.

**Comment by Eugen2525** [comment]
So, all the mentioned conditions are satisfied. In fact i run the script in the readme file. Python version is 3.5.2

I do not know what else could be the cause...Hence an issue raised

**Comment by aryamccarthy** [comment]
Can you show the exact command you’re using to run the script? Also show the output of `ls`.

**Comment by Eugen2525** [comment]
Here you are:

```
(base) System-Product-Name:/home/SIGMORPHON/2019/evaluation$ ls
evaluate_2019_task1.py  --output   --reference                 sample_2019_task1_output.txt  sample_2019_task2_output.txt
evaluate_2019_task2.py  README.md  sample_2019_task1_gold.txt  sample_2019_task2_gold.txt

(base) System-Product-Name:/home/SIGMORPHON/2019/evaluation$ python3 evaluate_2019_task2.py --reference sample_2019_task2_gold.txt --output sample_2019_task2_output.txt

```

**Comment by aryamccarthy** [comment]
Ah. This problem can be fixed by updating to Python 3.6.

See [this StackOverflow discussion](https://stackoverflow.com/questions/42694112/when-using-pathlib-getting-error-typeerror-invalid-file-posixpathexample-t): `open` isn't designed to work with `pathlib.Path` yet.
